index.html

```diff
@@ -249,25 +249,25 @@ <h1>Important Terms</h1>
 	<div>
 		<p>While some terms are defined in place, the following definitions are used throughout this document. </p>
 		<dl class="termlist">
-			<dt><dfn data-export=""  data-lt="accessibility api|accessibility apis">Accessibility <abbr title="Application Programming Interface">API</abbr></dfn></dt>
+			<dt><dfn data-lt="accessibility api|accessibility apis">Accessibility <abbr title="Application Programming Interface">API</abbr></dfn></dt>
```

Comment:
This would break all the other specs in the ARIA suite. Core-AAM for example references this definition 12 times. Other specs (Accname, dpub-aria, dpub-aam etc.) all reference this too. How would they then reference this term?